.github/scripts/release.sh

```diff
@@ -1,57 +1,312 @@
 #!/bin/bash
+set -euo pipefail
+IFS=$'\n\t'
 
-if ! mvn --version &> /dev/null
-then
-    echo "Maven is not installed or it is not added to the path!"
+printNewSection() {
+  echo
+  echo
+}
+
+printHelp() {
+  printNewSection
+  echo 'HELP'
+  echo '    This script can do the following things based on your commit that you checked out,'
+  echo '    and it will lead you through all steps:'
+  echo
+  echo '    Release'
+  echo '        Create a version tag with the current corresponding pom.xml version.'
+  echo '        This tag is than pushed to a selected remote repository.'
+  echo '    Setup'
+  echo '        Request a new version. This version is than updated in the pom.xml file.'
+  echo '        The CHANGELOG.md file will be updated and the timestamp of the previous release is added.'
+  echo '        Than the changes are pushed to a selected remote repository and branch.'
+  echo '        Than a PullRequest is created for the selected remote repository and branch.'
+  echo
+  echo '    A value in brackets before a input value indicates a default value if no input was given.'
+}
+
+deletePreviousLines() {
+  for (( i=0; i<$1; ++i)); do
+    echo -en '\033[1A\033[2K'
+  done
+}
+
+goToRootDirectory() {
+  CURRENT_PATH="$(pwd)"
```

Comment:
There is a "convention" that only constants are written in SHOUTY_CAPS and normal variables being in snake_case. This helps preventing accidental overwrites of environment variables, but should not cause any problems in this script.

As this script is consistent right now I do not require this to be changed, but still wanted to mention it.